### PR TITLE
Adding ActivityLogAlerts and ActionGroups API calls, PATH operations, and the corresponding unit tests

### PR DESCRIPTION
**Pull request by gucalder**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
   <PropertyGroup>
     <Description>Microsoft Azure Monitor Library</Description>
-    <VersionPrefix>0.16.0-preview</VersionPrefix>
+    <VersionPrefix>0.16.1-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/SDKs/Monitor/Management.Monitor/Microsoft.Azure.Management.Monitor.csproj]: @gucalder you have redundant tags, please remove versionPrefix and move the Version under the top PropertyGroup

File: src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <Description>Monitor.Tests Class library</Description>
     <PackageId>Monitor.Tests</PackageId>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <TargetFrameworks>net452</TargetFrameworks>
```
Comment (shahabhijeet) [inline src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj]: @gucalder any reason you want your tests to be targeting .NET 452?
If this was a mistake please remove

File: src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.5.0-preview" />
-    <ProjectReference Include="..\Management.Search\Management.Search.csproj" />
+    <ProjectReference Include="..\Management.Search\Microsoft.Azure.Management.Search.csproj" />
```
Comment (shahabhijeet) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: @gucalder do you have any changes for the Search management SDK? Or was this is a mistake.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj (hunk 3 above)
Comment (gucalder) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: Definitely a mistake. I do not remember changing this file.

File: src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj (hunk 2 above)
Comment (gucalder) [inline src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj]: This is odd, as soon as I remove .Net 452 from this project file, VS builds, but it does not detect any unit test.

Comment (gucalder) [inline src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj]: BTW: I can only build from VS 2017, not from the command line, nor from the root of the repo.
Is that expected?
Do I need to use any special argument to build from the command line?

File: src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj (hunk 3 above)
Comment (shahabhijeet) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: @gucalder vs17Dev is not the current dev branch. psSdkJson6 is the current dev branch.

Comment (brjohnstmsft) [inline src/SDKs/Search/Management/Search.Management.Tests/Search.Management.Tests.csproj]: @gucalder Please revert Search-related changes

**Review by bitsforthought — CHANGES_REQUESTED** [review]

**Pull request closed without merging**